### PR TITLE
Edit UserGuide and DeveloperGuide and Ui

### PR DESCRIPTION
**Pull request by Zhi-You**
part of  #61 
Enhance User Guide with new images and documentation.

Edited Ui too to make it more user friendly.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-# Developer Guide
+# Developer Guide for E-Duke-8
+
+- Table of Contents {:toc}
```
Comment (kstonekuan) [inline docs/DeveloperGuide.md:3]: Thanks for putting this placeholder, will try to get it done soon

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,102 +1,170 @@
...
+- Table of Contents {:toc}
 
 ## Introduction
 
-E-Duke-8 helps CS2113/T students learn and understand software engineering and OOP principles through a gamified
-platform and enhances their learning experience. It also consolidates key concepts for easy revision.
+E-Duke-8 helps CS2113/T students **learn and understand software engineering and OOP principles** through a gamified
+platform and enhances their learning experience. 
+
+It is a desktop application where CS2113/T students can attempt bite-sized quizzes, through the **Command Line Interface (CLI)**, to test their understanding of the concepts taught, and serves to consolidate key concepts for easy revision.
```
Comment (kstonekuan) [inline docs/UserGuide.md:10]: Nice formatting

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,102 +1,170 @@
...
 
-### Viewing product Information : `about`
+### Viewing E-Duke-8 information : `about`
+
+Shows a message explaining what E-Duke-8 is about and your benefits of using it.
+
+Example of usage:
+```
+about
+```
```
Comment (kstonekuan) [inline docs/UserGuide.md:54]: Good idea to make the usage clearer

**Review by kstonekuan — APPROVED** [review]

File: src/main/java/seedu/eduke8/ui/Ui.java [hunk 4]
```diff
@@ -48,6 +50,12 @@
...
 
     public String getInputFromUser() {
+        System.out.print("Enter your command or 'help': ");
+        return SCANNER.nextLine();
+    }
+
+    public String getQuizInputFromUser() {
+        System.out.print("Enter your answer or get a hint ('hint'): ");
         return SCANNER.nextLine();
     }
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/ui/Ui.java:60]: Ah this makes it look a lot cleaner and easier to understand! Thanks!

**Pull request merged**